go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
 	github.com/aws/aws-sdk-go v1.38.19
 	github.com/coreos/go-systemd/v22 v22.3.2
-	github.com/gardener/etcd-druid v0.6.1-0.20211022081522-071746e9d0bd
-	github.com/gardener/gardener v1.36.0
+	github.com/gardener/etcd-druid v0.7.0
+	github.com/gardener/gardener v1.38.0
```

Comment:
Optional

```suggestion
	github.com/gardener/gardener v1.38.1
```

Reply:
Is there something relevant in it? Otherwise, I'd propose to wait for the 1.39 revendoring PR.

Reply:
There is https://github.com/gardener/gardener/pull/5194. But of course such version bump (from v1.38.0 to v1.38.1) can be done in separate PR, that's why I also wrote it as optional. Let's then consider it for another PR.